wom/src/main/scala/wom/callable/TaskDefinition.scala

```diff
@@ -39,9 +39,13 @@ sealed trait TaskDefinition extends Callable {
     // a list of all files generated by `writeFile`.
     import TaskDefinition.instantiatedCommandMonoid
 
-    val fullInstantiatedCommand: ErrorOr[InstantiatedCommand] = commandTemplate.toList.foldMap(_.instantiate(mappedInputs, functions, valueMapper, runtimeEnvironment))
+    // Just the raw command parts, no separators.
+    val rawParts = commandTemplate.toList.map(_.instantiate(mappedInputs, functions, valueMapper, runtimeEnvironment))
+    // Add separators command parts and monoid smash down to one instantiated command.
```

Comment:
No offense but `Monoid Smash!` is the lamest super-hero move ever

Reply:
Don't make the monoid angry. You wouldn't like it when it's angry.